src/Microsoft.Diagnostics.Monitoring.EventPipe/Configuration/MetricSourceConfiguration.cs

```diff
@@ -30,8 +30,10 @@ public sealed class MetricEventPipeProvider
 
     public sealed class MetricSourceConfiguration : MonitoringSourceConfiguration
     {
+        public const string SharedSessionId = "SHARED";
+
         private readonly IList<EventPipeProvider> _eventPipeProviders;
-        public string SessionId { get; private set; }
+        public string UniqueSessionId { get; private set; }
```

Comment:
This is for uniquely identifying instances of a SHARED session - this is currently being used in cases where a SHARED session is rejected due to having arguments that don't match. Without having a unique identifier, all SHARED sessions would react to the `MultipleSessionsConfiguredIncorrectlyError` message.

Reply:
This has been renamed to ClientId